src/software/jetson_nano/display/robot_ui.py

```diff
@@ -0,0 +1,154 @@
+import sys, time, redis
+import digitalio
+import board
+from PIL import Image, ImageDraw, ImageOps
+import adafruit_rgb_display.st7735 as st7735
+
+sys.path.append("./screens/")
+sys.path.append("./lcd_user_interface/")
+sys.path.append("./rotary_encoder/")
+from lcd_user_interface import LcdDisplay
+from rotary_encoder import RotaryEncoder
+from home_screen import HomeScreen
+from menu_screen import MenuScreen
+from wheels_screen import WheelsScreen
+from chip_and_kick_screen import ChipAndKickScreen
+
+# Pins for Rotary Encoder
+BUTTON_PIN = "DAP4_DOUT"  # BOARD 35, TEGRA_SOC: 'DAP4_FS'
+PIN_1 = "GPIO_PE6"  # BOARD 40, TEGRA_SOC: 'DAP4_DOUT'
+PIN_2 = "DAP4_FS"  # BOARD 33, TEGRA_SOC: 'GPIO_PE6'
+
+# These keys indicate how to handle return values
+status_codes = {"none": 0, "change screen": 1, "edit": 2, "update redis": 3}
```

Comment:
this might work better as a class, and nit maybe rename to `LCDActions` or `ScreenActions`
~~~
class StatusCodes:
    NONE = 0
    CHANGE_SCREEN = 1
    EDIT_SCREEN = 2
    UPDATE_REDIS = 3
~~~

Reply:
done